### ISSUE TITLE
Job master cannot work after high concurrency request

### ISSUE THREAD

**Issue by maobaolong** [Alluxio/alluxio#13878]
**Alluxio Version:**

Master branch, hash is 6469555115

**Describe the bug**

Lots of log in the `job_master.log` like this

```
2021-08-02 20:26:50,651 INFO  plan.PlanCoordinator (PlanCoordinator.java:start) - Starting job Id=1627566478229 Config=EvictConfig{blockId=9001110601728, replicas=20}
2021-08-02 20:26:50,653 INFO  plan.PlanCoordinator (PlanCoordinator.java:start) - Starting job Id=1627566478230 Config=EvictConfig{blockId=10100639006720, replicas=1}
2021-08-02 20:26:50,661 INFO  plan.PlanCoordinator (PlanCoordinator.java:start) - Starting job Id=1627566478231 Config=EvictConfig{blockId=9013995503616, replicas=10}
2021-08-02 20:26:51,047 WARN  util.JvmPauseMonitor (JvmPauseMonitor.java:detectPause) - JvmPauseMonitor-alluxio-master-1_20003: Detected pause in JVM or host machine (eg GC): pause of approximately 140140428ns.
GC pool 'PS MarkSweep' had collection(s): count=1 time=386ms
2021-08-02 20:26:51,066 INFO  plan.PlanCoordinator (PlanCoordinator.java:start) - Starting job Id=1627566478232 Config=EvictConfig{blockId=10139293712384, replicas=1}
2021-08-02 20:26:51,413 INFO  plan.PlanCoordinator (PlanCoordinator.java:start) - Starting job Id=1627566478233 Config=EvictConfig{blockId=10173653450752, replicas=1}
2021-08-02 20:26:51,455 INFO  plan.PlanCoordinator (PlanCoordinator.java:start) - Starting job Id=1627566478234 Config=EvictConfig{blockId=11363376168960, replicas=5}
2021-08-02 20:26:51,458 INFO  plan.PlanCoordinator (PlanCoordinator.java:start) - Starting job Id=1627566478235 Config=EvictConfig{blockId=11367671136256, replicas=37}
2021-08-02 20:26:51,475 INFO  plan.PlanCoordinator (PlanCoordinator.java:start) - Starting job Id=1627566478236 Config=EvictConfig{blockId=14704776839168, replicas=85}
```

**To Reproduce**

We have 1000 alluxio-fuse to support client side read, keep any configurations as default, we see 85 replicas while we set replication.max to 3, and ReplicationChecker will found the blocks over replicated and send evict command to job master.

Job master have a lots of job to do...

but after our workload stopped, no any request outside alluxio cluster, but job_master.log shown that job server are receiving lots of create evict job request, guest it from alluixo-master.

I try to find some traces by job cli, but it cannot be connected. Luckily, the restful api is ok, I send the following request,

http://localhost:51346/api/v1/master/job/list

and find there are 21169 job, and they are completed job, the count are 21169 all the time.

**Expected behavior**

Job service can works with high concurrency request scenario.

**Urgency**

High

**Additional context**
No

**Comment by yuzhu** [comment]
this is a known issue and was fixed by https://github.com/Alluxio/alluxio/pull/13859, please update with that patch.

**Comment by maobaolong** [comment]
@yuzhu I'm very excited to hear this good news, thank you to let me know this, will update our branch.

**Comment by beinan** [comment]
@yuzhu can we close this one?

**Comment by yuzhu** [comment]
fixed by #13859
